Mod2/Assessments/Week3Practical.md

```diff
@@ -0,0 +1,23 @@
+# Week 3 Assessment
+
+Follow these steps to setup the assessment:
+1. Create a new database named `flight_db` using pgAdmin.
+2. Copy [this script](flight_db.sql) and paste it into the query tool to insert records into your database.
+3. Run the following `SELECT` queries individually to get an understanding of the data:
+> `SELECT * FROM airlines;`
+> `SELECT * FROM flights;`
+
+Please provide the QUERY (not the answer) that returns each of the following:
```

Comment:
I wonder if we should add explicit instructions to encourage students to try running the queries as they are writing them?